raiden/tasks.py

```diff
@@ -50,6 +51,30 @@ def check_version():
             gevent.sleep(CHECK_VERSION_INTERVAL)
 
 
+def check_gas_escrow(raiden):
+    """ Check periodically for gas escrow in the account """
+    while True:
+        has_enough_balance, estimated_required_balance = gas_escrow.has_enough_gas_escrow(
+            raiden,
+            channels_to_open=0,
+        )
+        estimated_required_balance_eth = Web3.fromWei(estimated_required_balance, 'ether')
+
+        log.debug('Estimated safe balance', required_wei=estimated_required_balance)
+        if not has_enough_balance:
+            click.secho(
+                (
+                    'WARNING\n'
+                    "Your account's balance is below the estimated safe amount of "
+                    f'{estimated_required_balance_eth} eth. This may lead to a loss of '
+                    'funds. Please add funds to your account as soon as possible.'
```

Comment:
loss of funds -> loss of funds because your account will be unable to perform on-chain transactions.

^ - That message needs to be as detailed as possible.